zng/docs/spec.md

```diff
@@ -349,6 +351,32 @@ previously defined type, the appropriate typedef control code must
 be re-emitted
 (and note that the typedef may now be assigned a different ID).
 
+### 3.1.3 Compressed Message Block
+
+Following a header byte of 0x86 is a compressed message block.  Such a
+block comprises a compressed sequence of control and value messages.
+The sequence can include any message except for end-of-stream markers
+and compressed message blocks.
+
+A compressed message block is encoded as follows:
+```
+------------------------------------------------------------------------------
+|0x86|<format><uncompressed-length>|<compressed-length>|<compressed-messages>|
+------------------------------------------------------------------------------
+
+```
+where 
+* `<format>`, a `uvarint`, identifies the algorthim used to compress the
+  message sequence
+* `<uncompressed-length>`, a `uvarint`, is the length in bytes of the
+  uncompressed message sequence
+* `<compressed-length>`, a `uvarint`, is the length in bytes of `<compressed-messages>`
+* `<compressed-messages>` is the compressed message sequence
+
```

Comment:
Can you doc that if compressed length is equal to uncompressed, the messages are not compressed?

Reply:
`<compressed-messages>` is always compressed, regardless of the values of `<uncompressed-length>` of `<compressed-length>`.

Can you elaborate on what you're thinking here?

Reply:
Ah, you're thinking about https://github.com/brimsec/zq/pull/1050/files/c4caa5a4457ea4a021cddad1d1f531d11adbc22e#diff-0b839f9afb00c1beb6d0adccd50777f1R226, right? That's from an earlier iteration and needs to be removed.